pkg/recipes/configloader/environment.go

```diff
@@ -101,14 +101,16 @@ func getConfiguration(environment *v20231001preview.EnvironmentResource, applica
 		return nil, ErrUnsupportedComputeKind
 	}
 
-	providers := environment.Properties.Providers
-	if providers != nil {
-		if providers.Aws != nil {
-			config.Providers.AWS.Scope = to.String(providers.Aws.Scope)
-		}
-		if providers.Azure != nil {
-			config.Providers.Azure.Scope = to.String(providers.Azure.Scope)
-		}
+	env, err := environment.ConvertTo()
```

Comment:
Let's add a comment here that it converts it to internal data model since it is not as obvious from the function name.